components/contracts/ActionContainer.tsx

```
@@ -93,98 +93,100 @@ const ActionContainer: FC<ActionContainerProps> = ({
+                onPress={() => {
+                  {
+                    try {
+                      sendSystemMessage(
+                        contract.streamChannelId,
+                        "Let's end the contract"
+                      );
+                      Toaster.success("Sent message to the brand");
+                    } catch (e) {
+                      console.log(e);
+                    }
+                  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)